packages/react/src/providers/treasure.tsx

```diff
@@ -190,10 +192,32 @@ export const TreasureProvider = ({
     [tdk.user.me, onConnect],
   );
 
-  const { isUsingTreasureLauncher, openLauncherAccountModal } = useLauncher({
+  const onWalletComponentsUpdated = useCallback(
```

Comment:
Does this function need to be in the Treasure provider? Seems like it could be done within `useLauncher` itself

Reply:
Moved this to `useLauncher`